source/features/bugs-tab.tsx

```diff
@@ -24,16 +24,40 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const cacheKey = (): string => 'bugs-label' + ':' + getRepo()!.nameWithOwner;
 const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+	if (!await cache.get(cacheKey())) {
+		const {repository} = await api.v4(`
+			repository() {
+				labels(query: "bug", first: 10) {
+					nodes {
+						name
+						issues(states: OPEN) {
+							totalCount
+						}
+					}
+				}
+			}
+		`);
+
+		const {name: bugLabel, issues} = [...repository.labels.nodes].find(label => [':bug: bug', 'bug', 'confirmed-bug', 'type: bug'].includes(label.name.toLowerCase()));
+		void cache.set(cacheKey(), bugLabel.includes(':') ? `"${bugLabel}"` : bugLabel ?? false);
+		return issues?.totalCount ?? 0;
+	}
+
+	const {repository} = await api.v4(`
+		repository() {
+			label(name: "${(await cache.get(cacheKey())).replaceAll('"', '')}") {
```

Comment:
I like this api more than the search() API, it's shorter and more correct. Are they equivalent? If so, let's use `label()` instead of `search()`

Reply:
Only thing is that its case-sensitive. So if someone changes from "bug" to "Bug" it will not work

Reply:
> And definitely not 2 separate API calls

Why should we query for all labels every time.

Each time we will only use ONE api call.

The first time will load all labels the other times it will ONLY load a single one.

Reply:
Looking good! Proceed 😁

Reply:
Ok the only issue I am having is the `isBugPage` encoding with `"` But I will try to work on it myself

Reply:
> Only thing is that its case-sensitive. So if someone changes from "bug" to "Bug" it will not work


That's the case for all of our cached content. If the content changes, they'll not see the right data for a while.